eng/common/scripts/stress-testing/deploy-stress-tests.ps1

```diff
@@ -22,6 +22,7 @@ $FailedCommands = New-Object Collections.Generic.List[hashtable]
 
 . (Join-Path $PSScriptRoot "../Helpers" PSModule-Helpers.ps1)
 Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
+Install-ModuleIfNotInstalled "az" "6.4.0" | Import-Module
```

Comment:
We actually have a helper script just for this :) https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/Import-AzModules.ps1. However the script essentially does what you are doing here so I leave it to you how you want to use it.

Reply:
I figured just to be consistent with the other module install calls (and avoid double sourcing), I'd do it this way.